sunpy/io/special/asdf/tags/coordinates/frames.py

```diff
@@ -11,24 +11,24 @@
 __all__ = ['SunPyCoordType']
 
 
-SCHEMA_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__),
+SCHEMA_PATH = Path(__file__).parent.joinpath(
                                            '..', '..',
                                            'schemas',
                                            'sunpy.org',
-                                           'sunpy'))
+                                           'sunpy').resolve()
```

Comment:
We must be able to clean up this file path.

Reply:
sorry i didnt understand, what do you mean exactly